src/main/native/org/apache/commons/crypto/OpenSslInfoNative.c

```diff
@@ -41,7 +41,10 @@
 #ifdef UNIX
 static unsigned long (*dlsym_SSLeay) (void);
 static char * (*dlsym_SSLeay_version) (int);
-static void *openssl;
+static unsigned long (*dlsym_OpenSSL_version_num) (void);
+static char * (*dlsym_OpenSSL_version) (int);
+static void *openssl_old, *openssl_new;
```

Comment:
I am not a big fan of *_old and *_new - can you choose more descriptive variables names?